src/string-class.js

```diff
@@ -35,8 +35,8 @@ String.prototype.isQuestion = function isQuestion() {
 
 String.prototype.words = function words() {
   /* creates an array of the string*/
-  const boundary = /\W+/;
-  return boundary.test(this) ? this.split(boundary) : [this];
+  const testCase = /^\w/;
+  return testCase.test(this) ? this.trim().split(' ') : [];
 };
```

Comment:
**Code Quality - Critical**

Remove trailing spaces  to avoid having them considered to be words